src/services/rsvp/rsvp-router.ts

```diff
@@ -0,0 +1,174 @@
+import { Request, Response, Router } from "express";
+import { StatusCode } from "status-code-enum";
+import { strongJwtVerification } from "../../middleware/verify-jwt.js";
+import { JwtPayload } from "../auth/auth-models.js";
+import { hasElevatedPerms } from "../auth/auth-lib.js";
+import { DecisionStatus, DecisionResponse, DecisionInfo } from "../../database/decision-db.js";
+import Models from "../../database/models.js";
+
+const rsvpRouter: Router = Router();
+
+/**
+ * @api {get} /rsvp/:USERID/ GET /rsvp/:USERID/
+ * @apiGroup rsvp
+ * @apiDescription Check RSVP decision for a given userId, provided that the current user has elevated perms
+ *
+ *
+ * @apiSuccess (200: Success) {string} userId
+ * @apiSuccess (200: Success) {string} User's applicatoin status
+ * @apiSuccess (200: Success) {string} User's Response (whether or whether not they're attending)
+ * @apiSuccess (200: Success) {string} Reviwer
+ * @apiSuccess (200: Success) {boolean} Whether email has been sent
+ * @apiSuccessExample Example Success Response:
+ * 	HTTP/1.1 200 OK
+ *	{
+ *      "userId": "github0000001",
+ *      "status": "ACCEPTED",
+ *      "response": "PENDING",
+ *      "reviewer": "reviewer1",
+ *      "emailSent": true
+ * 	}
+ *
+ * @apiUse strongVerifyErrors
+ */
+rsvpRouter.get("/:USERID", strongJwtVerification, async (req: Request, res: Response) => {
+    const userId: string | undefined = req.params.USERID;
+
+    const payload: JwtPayload = res.locals.payload as JwtPayload;
+    //Redirects if caller doesn't have elevated perms
```

Comment:
On second glance, I'm not sure if we should even be redirecting at all. For most of the other failures of this, we given a `Forbidden` error.

Not sure what the purpose of directing to the root of the API is?